recipe/build.sh

```diff
@@ -1,8 +1,10 @@
-#!/bin/bash -e
+#!/bin/bash
```

Comment:
agreed to this change, but you should be aware that the she-bang isn't used during conda-build anyway, as it rewrites the scripts ....

Reply:
Ah, that must be why `set` was working but editing the flags here wasn't. This change was just a side-effect from some test changes.

Reply:
I've put `set -ex`, it should exit if any line fails